testing/ostest/Makefile

```diff
@@ -97,6 +97,14 @@ endif
 ifeq ($(CONFIG_SCHED_WORKQUEUE),y)
 CSRCS += wqueue.c
 endif
+
+ifeq ($(CONFIG_PRIORITY_INHERITANCE),y)
+CSRCS += prioinherit.c
+endif
+
+ifneq ($(CONFIG_ARCH_SIM),y)
```

Comment:
why check sim?

Reply:
I've done more tests and found out that priority inheritance test does not work with `CONFIG_SIM_WALLTIME_SLEEP=y`, but only works with `CONFIG_SIM_WALLTIME_SIGNAL=y`. @xiaoxiang781216 do you know what may be the reason?
I will update the test case and include it unconditionally, but `sim:ostest` should be updated to use `CONFIG_SIM_WALLTIME_SIGNAL=y` to get this test working

Reply:
let's list this as a know issue before we can fix it.